### PR TITLE
✨ Feat: Workspace Selector Menu

### PR DESCRIPTION
**Pull request by kenjinp**
Select your orgs and projects from this menu!

### CODE REVIEW

File: src/shared/components/Header/WorkspaceSelectorMenu/WorkspaceSelectorContainer.tsx [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+
+const mapStateToProps = (state: RootState) => ({
+  displayPerPage: state.uiSettings.pageSizes.workspaceSelectorMenuSize,
+  orgs: (state.nexus && state.nexus.orgs) || {
+    isFetching: false,
+    data: null,
+    error: null,
+  },
+  activeOrg:
+    (state.nexus &&
```
Comment (mdupont) [inline src/shared/components/Header/WorkspaceSelectorMenu/WorkspaceSelectorContainer.tsx]: These nested checks are so ugly... I'm far from being a proponent of adding yet another dependency, but we may benefit from using [lodash.get](https://www.npmjs.com/package/lodash.get), which is self-contained, to make this easier to read and less brittle.

File: src/shared/components/Header/WorkspaceSelectorMenu/WorkspaceSelectorMenuButton.tsx [hunk 2]
```diff
@@ -0,0 +1,30 @@
+import * as React from 'react';
+import { Button, Popover } from 'antd';
+import { WorkspaceSelectorContainerProps } from './workspaceSelectorContainer';
+import { Organization, Project } from '@bbp/nexus-sdk';
+import WorkspaceSelectorMenu from './WorkspaceSelectMenu';
+
+const getButtonLabel = (
+  activeOrg: Organization | null,
```
Comment (mdupont) [inline src/shared/components/Header/WorkspaceSelectorMenu/WorkspaceSelectorMenuButton.tsx]: Any reason why it's not `activeOrg?: Organization`

Comment (kenjinp) [reply src/shared/components/Header/WorkspaceSelectorMenu/WorkspaceSelectorMenuButton.tsx]: I just like null better. I know I put it there

File: src/shared/components/Header/WorkspaceSelectorMenu/WorkspaceSelectorContainer.tsx (hunk 1 above)
Comment (kenjinp) [reply src/shared/components/Header/WorkspaceSelectorMenu/WorkspaceSelectorContainer.tsx]: we have a getProp function in the utils folder that can be used for this

**Pull request closed without merging**